packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/08-permission-requests.mdx

```diff
@@ -3,4 +3,368 @@ title: Permission Requests
+  - `yarn add @stream-io/video-react-sdk`
+- [Initialize the SDK](../../guides/client-auth) by passing in your API key, token and user information:
+
+:::note
+For the token generation, you can use our [Token Generator](https://getstream.io/chat/docs/token_generator/).
+:::
+
+## Start point
+
+Let's start with a simple call UI where we create an `audio_room` call.
+
+```tsx
+import {
+  useCreateStreamVideoClient,
+  StreamVideo,
+  StreamCall,
+  CallParticipantsView,
+  CallControls,
+  useCall,
+  StreamTheme,
+} from '@stream-io/video-react-sdk';
+import '@stream-io/video-react-sdk/dist/css/styles.css';
+
+export default function App() {
+  const client = useCreateStreamVideoClient({
+    apiKey: '<API Key>',
+    tokenOrProvider: '<token provider>',
+    user: {
+      id: '<user id>',
+    },
+  });
+
+  return (
+    <StreamTheme>
+      <StreamVideo client={client}>
+        <StreamCall
+          callId={'test-call-permission-requests'}
+          callType={'audio_room'}
+          data={{ create: true }}
+          autoJoin
+        >
+          <CallUI />
+        </StreamCall>
+      </StreamVideo>
+    </StreamTheme>
+  );
+}
+
+const CallUI = () => {
+  const call = useCall();
+
+  return (
+    <div>
+      <PermissionRequestNotifications />
+      <CallParticipantsView call={call} />
+      <CallControls />
+      <PermissionRequests />
+    </div>
+  );
+};
+
+const PermissionRequests = () => {};
+
+const PermissionRequestNotifications = () => {};
+```
+
+:::note
+
+Permission requests are only relevant for [call types](../../guides/10-configuring-call-types/) that have roles where a participant can't publish their audio/video/screen-share by default.
+
+:::
+
+In this guide, we will implement the `PermissionRequests` component to send permission requests and the `PermissionRequestNotifications` to receive them.
+
+To follow along with the guide, don't forget to [turn off backstage mode](../../guides/call-lifecycle/#backstage-mode) so others can join:
+
+```typescript
+let call: Call;
+
+call.goLive();
+```
+
+## Request permission
+
+In this step we will implement the `PermissionRequests` component to send permission requests.
+
+For readability the code snippet only contains the `PermissionRequests` implementation, the [full example](#finished-example) is available at the end of the guide.
+
+```tsx
+const PermissionRequests = () => {
+  const call = useCall();
+  const canSendAudio = useHasPermissions(OwnCapability.SEND_AUDIO);
+  const canSendVideo = useHasPermissions(OwnCapability.SEND_VIDEO);
+  const canShareScreen = useHasPermissions(OwnCapability.SCREENSHARE);
+
+  if (!call) {
+    return null;
+  }
+
+  const shouldDisplayButton = (capability) => {
+    return (
+      call.permissionsContext.canRequest(capability) &&
+      !call.permissionsContext.hasPermission(capability)
+    );
+  };
+
+  const requestPermission = (capability) => {
+    call.requestPermissions({ permissions: [capability] });
```

Comment:
Permissions and capabilities are synonyms?

Reply:
Yes, they are called `permission` in one part of the API, and `OwnCapability` in the other